### PR TITLE
Handle changed dcos cert path in 1.10

### PR DESCRIPTION
**Pull request by benclarkwood**

### CODE REVIEW

File: sdk/bootstrap/main.go [hunk 1]
```diff
@@ -269,12 +269,19 @@ func installDCOSCertIntoJRE() {
...
 	certExists, err := isFile(certPath)
 	if !certExists || err != nil {
-		log.Printf("No $MESOS_SANDBOX/.ssl/ca.crt file found. Cannot install certificate. Error: %s", err)
-		return
+		log.Printf("No $MESOS_SANDBOX/.ssl/ca-bundle.crt file found. Error: %s. Checking pre 1.10 location", err)
+		certPath = filepath.Join(mesosSandbox, ".ssl", "ca.crt")
+		certExists, err = isFile(certPath)
+		if !certExists || err != nil {
+			log.Printf("No $MESOS_SANDBOX/.ssl/ca.crt file found. Error: %s.", err)
+			log.Printf("No CA Cert found in the sandbox. Cannot install certificate.")
```
Comment (nickbp) [inline sdk/bootstrap/main.go]: IMO these logs could be updated to clarify that this is expected on non-enterprise clusters

File: sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosCertInstaller.java [hunk 2]
```diff
@@ -20,7 +20,8 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(DcosCertInstaller.class);
 
     public static final String DEFAULT_JRE_KEYSTORE_PASS = "changeit";
-    public static final String CERT_PATH = ".ssl/ca.crt";
+    public static final String PRE_110_CERT_PATH = ".ssl/ca.crt";
```
Comment (nickbp) [inline sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosCertInstaller.java]: Can't hurt to add javadocs on these explaining the situation. Also it looks like none of these need to be public?

**Comment by benclarkwood** [comment]
@nickbp Comments addressed 👍

**Comment by benclarkwood** [comment]
Known flaky HDFS tests. Merging.

**Pull request merged**